### PR TITLE
Add task on plugging existing key/cert into Istio CA

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

File: _docs/tasks/security/plugin-ca-cert.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+We would like to use `root-cert.pem` as the root certificate for Istio workloads.
+
+When the Istio CA certificate is not the workloads' root certificate,
+the workload cannot validate the workload certificates directly from the root certificate.
+In this case, the workload needs a `cert-chain.pem` file to specify the chain of trust,
+which should include the certificates of all the intermediate CAs between the workloads and the root CA.
+
+The following command downloads the example files:
+```bash
+wget https://raw.githubusercontent.com/istio/auth/master/samples/plugin_ca_certs/ca-cert.pem
```
Comment (ldemailly) [inline _docs/tasks/security/plugin-ca-cert.md]: change to new location

File: _docs/tasks/security/plugin-ca-cert.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  ```
+
+* To remove all the secrets generated by Istio:
+  ```bash
+  kubectl delete secret --all
+  ```
+
+## Further reading
+
+* Read the Istio [CA arguments](https://github.com/istio/auth/blob/master/cmd/istio_ca/main.go).
```
Comment (ldemailly) [inline _docs/tasks/security/plugin-ca-cert.md]: change to new location

File: _docs/tasks/security/plugin-ca-cert.md (hunk 1 above)
Comment (myidpt) [reply _docs/tasks/security/plugin-ca-cert.md]: Done.

File: _docs/tasks/security/plugin-ca-cert.md (hunk 2 above)
Comment (myidpt) [reply _docs/tasks/security/plugin-ca-cert.md]: Done.

**Comment by myidpt** [comment]
@geeknoid Hi Martin, Seb and Yutong suggested that you are managing this repo. So do you know who should I contact to solve the presubmit error, or can we just ignore the error? It would be great if this PR can be merged this week. Thanks!

**Comment by ldemailly** [comment]
you need to fix the error:
- ./_site/docs/reference/config/mixer/adapters/index.html
  *  External link https://github.com/istio/istio/blob/master/mixer/doc/dev/adapters.md failed: 404 No error

**Comment by myidpt** [comment]
@ldemailly Fixing the error in https://github.com/istio/istio.github.io/pull/707

**Pull request merged**